test/N3Util-test.js

```diff
@@ -196,6 +196,16 @@ describe('Util', () => {
         expect(rdfs('label')).to.deep.equal(new NamedNode('http://www.w3.org/2000/01/rdf-schema#label'));
       });
 
+      it('should expand a NamedNode prefix', () => {
+        const rdfs = Util.prefix(new NamedNode('http://www.w3.org/2000/01/rdf-schema#'));
+        expect(rdfs('label')).to.deep.equal(new NamedNode('http://www.w3.org/2000/01/rdf-schema#label'));
+      });
+
+      it('should expand an empty literal prefix', () => {
+        const empty = Util.prefix(new Literal(''));
```

Comment:
Well, rather a literal with a URL in there… let's hope they never use it for this 😄

Reply:
I added this test because it breaks when using `||` but doesn't when using `??`. But I definitely agree this isn't the intended use of `Util.prefix` :sweat_smile: